Tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -340,6 +342,16 @@ public function testDeprecatedRenderListElement($expected, $value, array $option
         );
     }
 
+    public function buildUrl($url)
```

Comment:
* Call it `buildTwigLikeUrl`
* Make it `private`
* A a phpdoc
* On the PHP doc, specify that is a BC for PHP 5.3 and twig
* As @greg0ire said, add something like `NEXT_MAJOR: remove this method as PHP 5.3 support will be dropped`